src/FSharp.Core/reflect.fsi

```diff
@@ -765,6 +834,7 @@ module FSharpReflectionExtensions =
         /// <returns>The constructed union case.</returns>
         ///
         /// <example-tbd></example-tbd>
+        /// @@@@@ TBD: UnionCaseInfo contains a Type
```

Comment:
Could it be solved by marking the property on UnionCaseInfo ?
Because then, whenever UnionCaseInfo is retrieved/created, it will mark the type for not being trimmed as well.

Reply:
nice ... I think it might.

Reply:
Not necessary because UnionCaseInfo isn't user mutable anyway.